gr-azure-software-radio/include/azure_software_radio/difi_sink_cpp.h

```diff
@@ -12,51 +12,20 @@
 
 namespace gr {
   namespace azure_software_radio {
-
```

Comment:
can we have the docs in both places (yaml and source code) for now until we figure out what the issue is with cmake?

Reply:
but then it will show up doubled in the doc tab